internal/distro/rhel9/images.go

```diff
@@ -404,6 +412,15 @@ func edgeSimplifiedInstallerImage(workload workload.Workload,
 
 	rawImg.Filename = t.Filename()
 
+	if bpIgnition := customizations.GetIgnition(); bpIgnition != nil && bpIgnition.FirstBoot != nil && bpIgnition.FirstBoot.ProvisioningURL != "" {
+		rawImg.KernelOptionsAppend = append(rawImg.KernelOptionsAppend, "ignition.config.url="+bpIgnition.FirstBoot.ProvisioningURL)
+	}
+
+	// 92+ only
+	if kopts := customizations.GetKernel(); kopts != nil {
+		rawImg.KernelOptionsAppend = append(rawImg.KernelOptionsAppend, kopts.Append)
+	}
```

Comment:
These changes are made to the raw image when created for an installer but not for raw image builds.  Shouldn't this be done in the `NewOSTreeRawImage()` function instead so that it works for both raw image and simplified installer?

Reply:
since this is playing with customization and if I understand the new framework correctly, should I just copy this to `edgeRawImage` too?

Reply:
Good question.  At first I had the second option in mind but I think the first one is simpler.  Copy the same code to the `egeRawImage`.  I might simplify/unify the two in the future but I need to rework some other things first.